modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -89,12 +89,18 @@ struct DataQueue {
         GAPI_Assert(t == DESYNC);
     }
 
+    explicit DataQueue(tag t, cv::gapi::own::DropStrategy<Cmd>&& strategy)
+        : q(new cv::gimpl::stream::DesyncQueue(std::move(strategy))) {
```

Comment:
hmmm

Reply:
maybe you should create an object of a proper type based on TAG (which might have more than one value now?)